test/e2e-common.sh

```diff
@@ -159,6 +174,13 @@ function install_knative_serving_standard() {
   wait_until_pods_running knative-serving || return 1
   wait_until_pods_running istio-system || return 1
   wait_until_service_has_external_ip istio-system istio-ingressgateway
+
+  if (( INSTALL_MONITORING )) || [[ -n "${INSTALL_MONITORING_YAML}" ]]; then
```

Comment:
Now INSTALL_MONITORING_YAML will be empty unless you explicitly call install_knative_serving_standard with 4 parameters, or if you use --install-monitoring, so you don't have to check INSTALL_MONITORING here or in teardown.

Reply:
Removed.